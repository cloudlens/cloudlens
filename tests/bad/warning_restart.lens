stream (e in log.a) when (e) {
       print(1)
}

stream {
       print(e.message)
}

stream (e in log.a) when (e) {
       print(2)
}

{}

stream {
       print(e.message)
}

