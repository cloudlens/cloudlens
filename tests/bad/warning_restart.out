Warning: tests/bad/warning_restart.lens line 9, implicit restart of stream log.a!
Warning: tests/bad/warning_restart.lens line 15, implicit restart of stream log!
